python/baseline/remote.py

```diff
@@ -40,17 +42,24 @@ def predict(self, examples):
         :param examples: The input examples
         :return: The outcomes
         """
-        import requests
+        import httplib, urllib
```

Comment:
`httplib` doesn't exist in python3. We use six for python2/3 stuff.

We need this line `from six.moves.http_client import  HTTPConnection` to make it work on 2 and 3